HostDependencies.md

```diff
@@ -24,31 +24,66 @@ As dependencies and packages may be frequently changed, deprecated or updated th
 The basic build package on Ubuntu is the `build-essential` package. To install run:
 
 ```
-sudo apt-get update
-sudo apt-get install build-essential
+sudo apt update
+sudo apt install -y build-essential
 ```
 
 Additional base dependencies for building seL4 projects on Ubuntu include installing:
 ```
-sudo apt-get install cmake ccache ninja-build
-sudo apt-get install python-dev python-pip python3-dev python3-pip
-sudo apt-get install libxml2-utils ncurses-dev
-sudo apt-get install curl git doxygen device-tree-compiler
+sudo apt install -y \
+    cmake ccache ninja-build \
+    python-dev python-pip python3-dev python3-pip \
+    libxml2-utils ncurses-dev \
+    curl git doxygen device-tree-compiler
 ```
 
 To build for ARM targets you will need a cross compiler. In addition, to run seL4 projects on a simulator you will need `qemu`. Installation of these additional base dependencies include running:
 
 
 ```
-sudo apt-get install gcc-arm-linux-gnueabi g++-arm-linux-gnueabi
-sudo apt-get install gcc-aarch64-linux-gnu g++-aarch64-linux-gnu
-sudo apt-get install qemu-system-arm qemu-system-x86
+sudo apt install -y \
+    gcc-arm-linux-gnueabi g++-arm-linux-gnueabi \
+    gcc-aarch64-linux-gnu g++-aarch64-linux-gnu \
+    qemu-system-arm qemu-system-x86
 ```
 
 (you can install  the hardware floating point versions as well if you wish"
 ```
-sudo apt-get install gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf
+sudo apt install -y gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf
 ```
+
+####  Fedora 
```

Comment:
I'm not broadly opposed to putting in Fedora instructions, except that:

- no-one within our team uses Fedora, 
- our CI docker images are Debian based.

Given the similarity of packages between Fedora and Debian, I'm tempted to say that Fedora users could relatively easily infer what they need.
Happy to be convinced otherwise though.

Reply:
So, is there any progress of supporting more platform. There's more and more user in arch (Manjaro) or Fedora.